TemporaryEmail/source/info.plist

```diff
@@ -367,11 +375,13 @@ Because [Firefox does not support AppleScript](https://bugzilla.mozilla.org/show
 	</dict>
 	<key>variables</key>
 	<dict>
+		<key>notifications_enabled</key>
+		<string>true</string>
```

Comment:
I set it to `true` by default but I use `false` for me.

Reply:
I appreciate that, even though this feature won’t be accepted.